1-de-index.html

```diff
@@ -10,7 +10,7 @@
 mAltProgIcon: "Jamulus Icon"
 mTSlogan: "Musiziere online. Mit deinen Freunden. Kostenlos."
 mTGetStartedNow: "Jetzt Starten!"
-mHTMLDownloadNow: 'Erweitert: <a href="https://sourceforge.net/projects/llcon/files/latest/download" target="_blank" id="dld_frm_sf" rel="noreferrer">sofort herunterladen</a>'
+mHTMLDownloadNow: 'Sofort herunterladen für <a href="https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_win.exe" target="_blank" rel="noreferrer">Windows</a>, <a href="https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_mac.dmg" target="_blank" rel="noreferrer">Mac</a>, <a href="https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_ubuntu_amd64.deb" target="_blank" rel="noreferrer">Debian/Ubuntu</a> or <a href="https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0" target="_blank" rel="noreferrer">andere Plattformen</a>.'
```

Comment:
I think you should use the Jekyll variables here (see the wiki/en/\*Installation-For\* files

Reply:
Thanks, done.

Reply:
Using these variables broke the site because they are not expanded in front matter. :(
https://fettblog.eu/snippets/jekyll/liquid-in-frontmatter/

I chose a different approach in #366 now.